catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.osgi.framework.ServiceReference;
+
+import ddf.catalog.Constants;
+import ddf.catalog.transform.InputTransformer;
+import javafx.util.Pair;
+
+/**
+ * Provides information on available transformers
+ */
+@Service
+@Command(scope = CatalogCommands.NAMESPACE, name = "transformers", description = "Provides information on available transformers.")
+public class TransformersCommand extends CatalogCommands {
+
+    private static final String MIME_TYPE = "mime-type";
+    private static final String SCHEMA = "schema";
+
+    private static final String NOT_AVAILABLE = "N/A";
+    private static final String LINE = "------------------------";
+
+    HashMap<String, Pair<String, ArrayList<String>>> transformersProperties;
```

Comment:
❗️ Don't use `Pair` for this. That's a horrible misuse of a graphics library.

✏️ Look at [Guava's Multimap](https://google.github.io/guava/releases/21.0/api/docs/com/google/common/collect/Multimap.html) instead. 

What you have here is a `Map<String, Map<String, List<String>>>` where you want to constrain the inner map to a single entry. I suppose you could make a `Map<String, Map.Entry<String, List<String>>>` but that would be overkill. So, using a Multimap, you can create a `Map<String, Multimap<String>>` and work with it more easily.

Also, even though you're using these internally and not returning them, you should get in the habit of declaring things abstractly: e.g. `Map<...>` instead of `HashMap<...>`.